src/ProductConstructionService/ProductConstructionService.DependencyFlow/PullRequestUpdater.cs

```diff
@@ -138,17 +141,33 @@ public async Task<bool> UpdateAssetsAsync(
     public async Task<bool> ProcessPendingUpdatesAsync(SubscriptionUpdateWorkItem update)
     {
         _logger.LogInformation("Processing pending updates for subscription {subscriptionId}", update.SubscriptionId);
+        bool isCodeFlow = update.SubscriptionType == SubscriptionType.DependenciesAndSources;
 
         // Check if we track an on-going PR already
         InProgressPullRequest? pr = await _pullRequestState.TryGetStateAsync();
```

Comment:
Unintentional change?
```suggestion
        // Check if we track an on-going PR already
        InProgressPullRequest? pr = await _pullRequestState.TryGetStateAsync();
        bool isCodeFlow = update.SubscriptionType == SubscriptionType.DependenciesAndSources;
```